app/models/crop.rb

```diff
@@ -350,11 +350,7 @@ def name_already_exists(list_name, name)
   end
 
   def count_uses_of_property(col_name)
-    data = Hash.new(0)
-    plantings.each do |p|
-      data[p.send("#{col_name}")] += 1 if !p.send("#{col_name}").blank?
-    end
-    data
+    plantings.unscoped.where(crop_id: id).where.not("#{col_name}": nil).group(col_name).count
```

Comment:
Aaah, if you delete the #{ }, it becomes `col_name: nil` and `col_name` is interpreted as a symbol, not the name of a variable. I've replaced the colon with `=>`.